.github/workflows/release.yml

```diff
@@ -31,14 +31,26 @@ jobs:
         uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # pin@5.5.0
         with:
           go-version: '${{ env.GOLANG_VERSION }}'
+
       - name: Run unit tests
         run: make test-coverage
+
       - name: Build Helm Binaries
         run: |
           set -eu -o pipefail
 
-          make build-cross VERSION="${{ github.ref_name }}"
-          make dist checksum VERSION="${{ github.ref_name }}"
+      - name: Run Go Release
+        uses: goreleaser/goreleaser-action@v2
```

Comment:
Can we use the latest version (6.3.0) please, and pin to that specific release via hash (see an example `uses:` above)